Assignment3/src/com/metacube/get2019/Utility.java

```diff
@@ -0,0 +1,35 @@
+package com.metacube.get2019;
+
+/**
+ * The class has methods for
+ * some program based error handling 
+ * that is required
+ * @author Kritika
+ *
+ */
+public class Utility {
+	
+	/**
+	 * For no entity situation
+	 */
+	public void isValidate(){

```

Comment:
What is the purpose of this method?

Reply:
it is for the situation coming in ClassInfo class, when there are 0 students.